lib/semian/atomic_enum.rb

```diff
@@ -0,0 +1,39 @@
+require 'forwardable'
+
+module Semian
+  class AtomicEnum #:nodoc:
+    extend Forwardable
+
+    def_delegators :@integer, :execute_atomically, :transaction, :shared?, :destroy
+    private :shared?
+
+    def initialize(symbol_list, **_options)
```

Comment:
What interface? I don't understand.. Why would you give this thing a `name: 'bla'` parameter if the class is not doing anything with it?